CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 - [changed] Exposed `admin.firestore.DocumentReference` and
   `admin.firestore.DocumentSnapshot` types from the Admin SDK typings.
+- [changed] Upgraded Firestore dependency version to 0.11.2. This includes
```

Comment:
I'd recommend just linking to the Changelog here: https://github.com/googleapis/nodejs-firestore/releases/tag/v0.11.2

Reply:
Done